docs/reference/mapping/removal_of_types.asciidoc

```diff
@@ -429,8 +431,11 @@ POST _reindex
 
 Index creation, mappings and document APIs support the `include_type_name`
 option. When set to `false`, this option enables the behavior that will become
-default in 8.0 when types are removed. See some examples of interactions with
-Elasticsearch with this option turned off:
+default in 8.0 when types are removed. Since the default for `include_type_name` 
+is `false` starting with 7.0, you can omit using it after 7.0, but you can use
+it in case you want to temporarily force the legacy behaviour.
+
```

Comment:
May be we can phrase this paragraph simpler, removing the sentence "When set to `false`, this option enables the behavior that will become.."?

Reply:
Related to @mayya-sharipova's point, I think this whole  paragraph will need to be reworked a bit, since we have changed the plan significantly since it was written:
- It is only index/ mapping APIs that support this parameter now (not document APIs).
- It would be good to mention that this parameter is introduced in 6.7 with a default of `true`, and then will be defaulted to `false` in 7.0 and removed in 8.0. In particular, I think it is a little confusing to say 'When set to `false`, this option enables the behavior that **will become default in 8.0** when types are removed.'